### PR TITLE
Add supported operations of workflow definition

### LINKED ISSUE #14121: Add supported operations of workflow definition
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

Add supported operations of workflow definition

### Documentation Links

_No response_

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by rickchengx**
## Purpose of the pull request
* close: #14121 

## Verify this pull request

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

(or)

If your pull request contain incompatible change, you should also add it to `docs/docs/en/guide/upgrede/incompatible.md`

### CODE REVIEW

File: docs/docs/en/guide/project/workflow-definition.md [hunk 1]
```diff
@@ -128,6 +128,9 @@ The following are the operation functions of the workflow definition list:
 - **Timing Management:** The timing management page can edit, online or offline and delete timing.
 - **Delete:** Delete the workflow definition. In the same project, only the workflow definition created by yourself can be deleted, and the workflow definition of other users cannot be deleted. If you need to delete it, please contact the user who created it or the administrator.
 - **Download:** Download workflow definition to local.
+- **Copy:** Copy a new workflow based on the current workflow.
```
Comment (zhongjiajie) [inline docs/docs/en/guide/project/workflow-definition.md]: Could we add some notice to tell users can copy to both some project or to others project?
BTW, we should also tell user when they copy the workflow, the workflow name will and copy date as is post to compare between previous tasks

Comment (rickchengx) [reply docs/docs/en/guide/project/workflow-definition.md]: Thanks for your comment. I'll add this.

**Comment by rickchengx** [comment]
1. update the batch operations on the ui
image

2. remove duplicate text in `docs/docs/en/guide/project/workflow-definition.md`

**Comment by zhongjiajie** [comment]
> 1. update the batch operations on the ui
> 
> image
> 2. remove duplicate text in `docs/docs/en/guide/project/workflow-definition.md`

well done, BTW a little tips here, the next time when your pr scope changed you should also change the pr title. in this case we changed the scope from pure docs change to both docs and ui change

**Review by zhongjiajie — APPROVED** [review]

**Pull request merged**